src/coreComponents/constitutive/solid/PorousSolid.hpp

```diff
@@ -128,6 +145,50 @@ class PorousSolidUpdates : public CoupledSolidUpdates< SOLID_TYPE, BiotPorosity,
 //                                           volStrain );
   }
 
+  GEOSX_HOST_DEVICE
+  void smallStrainUpdateThermalSinglePhase( localIndex const k,
+                                            localIndex const q,
+                                            real64 const & initialFluidPressure,
+                                            real64 const & fluidPressure_n,
+                                            real64 const & fluidPressure,
+                                            real64 const & initialTemperature,
+                                            real64 const & temperature_n, 
+                                            real64 const & temperature,
+                                            real64 const ( &strainIncrement )[6],
+                                            real64 ( & totalStress )[6],
+                                            DiscretizationOps & stiffness ) const
```

Comment:
> I would be also in favor of unifying this function with the other `smallStrainUpdate` functions used in the poromechanical simulations, if possible. We could have one unique function that would compute total stress (+ derivatives wrt pressure, temperature, and strain) and body force (+ derivatives wrt strain and fluid density) and let the physics solvers compute the derivatives they actually need by using the chain rule. Something for another PR :)
> 
> That being said, I don't know if this is really possible in the fixed-stress scheme that you consider here, with the omission of the body force and the specific porosity update.

For now, it might be challenging to unify these `smallStrainUpdate` functions, mostly because the porosity update in the fixed stress thermoporomechanics has a correction term that is not needed in fully coupled one.